package.json

```diff
@@ -34,7 +35,8 @@
     "react-scripts": "^4.0.3",
     "redux": "^4.0.5",
     "redux-thunk": "^2.3.0",
-    "uuid": "^3.4.0"
+    "uuid": "^3.4.0",
+    "yup": "^0.32.9"
```

Comment:
I like this packages syntax and goals ( a real big fan of JOI ) I don't think it's worth having in the project. Dealing across cultures and countries there are to many permutations for most of our values to have a non simple regex pattern to use for validation.